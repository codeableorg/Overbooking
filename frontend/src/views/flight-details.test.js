// src/components/channel-list.test.js
import React from "react";
import { render } from "@testing-library/react";

import FlightDetails from "./flight-details";

test("FlightDetails component", () => {
  const { asFragment } = render(<FlightDetails />);
  expect(asFragment()).toMatchSnapshot();
});
