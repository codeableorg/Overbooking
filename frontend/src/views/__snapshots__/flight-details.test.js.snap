// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlightDetails component 1`] = `
<DocumentFragment>
  <div>
    <img
      class="css-ogmi5w-FlightDetails"
      src="airplane.png"
    />
    <ul
      class="css-1xrdt2g-FlightDetails"
    >
      <li>
        Fliht number: 311
      </li>
      <li>
        Capacity: 100
      </li>
      <li>
        Date: Monday, July 15th 2019
      </li>
      <li>
        Origin: LIMA
      </li>
      <li>
        Destination: SAO PAULO
      </li>
    </ul>
    <button
      class="css-549gpy-Button"
    >
      Start
    </button>
  </div>
</DocumentFragment>
`;
